doc/Makefile.am

```diff
@@ -50,6 +54,11 @@ if DOCUMENT_BUILDABLE
 dist-hook: html
 	@touch $(distdir)/build-stamp
 
+text: text-recursive
+text-am:
+markdown: markdown-recursive
+markdown-am:
```

Comment:
`make markdown` fails with "there is no rule to make markdown-am" like below if this target is omitted, so I have added this.

```
make[3]: ディレクトリ '/home/t-hashida/gitdir/groonga/doc/locale/en' に入ります
make[3]: *** ターゲット 'markdown-am' を make するルールがありません.  中止.
```

However, the `html` target don't define any task like `html-am` in `.am` files, but it is defined in `Makefile`s.
Who is define `html-am`? And it it good to define `markdown-am` and `text-am` manually in the `.am` files?